README.tpl.md

```diff
@@ -294,3 +294,16 @@ netrc-file = /PATH/TO/.netrc
 GITHUB_TOKEN=$YOUR_GITHUB_TOKEN niv add ...
 ```
 
+### How do I import a subpath of a source?
+
+In order to use the directory `dir` of a `my-package`, use the following
+pattern:
+
+``` nix
+let
+  sources = import ./nix/sources.nix;
+in sources.my-package + "/dir"
+```
+
+where `sources.my-package` is the root directory, and `+ "/dir"` appends the
```

Comment:
This is great! For some added clarity, I would suggest the following:

```suggestion
in this example, `sources.my-package` becomes `my-package`'s root directory, and `+ "/dir"` appends the
```

Reply:
That's great! I never know what's clear and what's not.